lib/Doctrine/DBAL/Types/ConversionException.php

```diff
@@ -99,4 +99,16 @@ static public function conversionFailedInvalidType($value, $toType, array $possi
             implode(', ', $possibleTypes)
         ));
     }
+
+    static public function conversionFailedSerialization($value, $format, $error, \Exception $previous = null)
```

Comment:
Not sure if we really need to introduce another method here. Maybe it would suffice the purpose by sticking to `conversionFailedFormat()`